doc/tools/docgen_cmd.py

```diff
@@ -13,6 +13,8 @@
 # version comparison
 from distutils.version import LooseVersion as V
 
+# List of workflows to ignore
+SKIP_WORKFLOWS_LIST = ['Workflow', 'CombinedWorkflow']
```

Comment:
use rather tuple as this list should no change
`SKIP_WORKFLOWS_LIST = ('Workflow', 'CombinedWorkflow')`